modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/WebDataFile_conf.pm

```diff
@@ -0,0 +1,160 @@
+
+package Bio::EnsEMBL::Production::Pipeline::PipeConfig::WebDataFile_conf;
+use strict;
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+use base ('Bio::EnsEMBL::Production::Pipeline::PipeConfig::Base_conf');
+use Bio::EnsEMBL::ApiVersion qw/software_version/;
+
+use Data::Dumper;
+
+sub default_options {
+    my $self = shift;
+    return {
+        %{$self->SUPER::default_options()},
+        species         => [],
+        division        => [],
+        antispecies     => [],
+        run_all         => 0, #always run every species
+        step             => [],  
+        genomeinfo_yml    => undef,
+        # Email Report subject
+        email_subject => $self->o('pipeline_name').'  pipeline has finished',
+        release => software_version()
+	};
+}
+
+sub pipeline_wide_parameters {
+    my $self = shift;
+    return { %{$self->SUPER::pipeline_wide_parameters()},
+        output_path       => $self->o('output_path'),
+        app_path          => $self->o('app_path'),
+        genomeinfo_yml    => $self->o('genomeinfo_yml')
+    };
+}
+
+sub pipeline_analyses {
+    my $self = shift;
+
+    return [
+        
+        { 
+          -logic_name  => 'GeneInfo_logic',
+          -input_ids  => [ {} ],
+          -module      =>  'Bio::EnsEMBL::Production::Pipeline::Webdatafile::GenomeInfoYml',
+          -parameters => { species => $self->o('species'),
+                           genomeinfo_yml => $self->o('genomeinfo_yml'), 
+                           run_all        => $self->o('run_all'),   
+                         },
+          -flow_into   =>  {  '2->A' => ['StepBootstrap'],
+                              '3->A'  => ['SpeciesFactory'],   
+                               # WHEN(
+                               #   '#genomeinfo_yml#' =>  ['StepBootstrap'],
+                               #    ELSE                  ['SpeciesFactory'],
+                               # ),
+                              'A->1' => ['email_notification'],   
+                           }
+
+        }, 
+
+        {
+            -logic_name => 'SpeciesFactory',
+            -module     =>
+                'Bio::EnsEMBL::Production::Pipeline::Common::SpeciesFactory',
+            #-input_ids  => [ {} ], # required for automatic seeding
+            -parameters => { species => $self->o('species'),
+                antispecies          => $self->o('antispecies'),
+                division             => $self->o('division'),
+                run_all              => $self->o('run_all') 
+             },
+            -flow_into  => { '2' => [ 'StepBootstrap' ],
+                             #'A->1' => ['email_notification']
+                           },
+             
+        },
+        {    
+            -logic_name => 'StepBootstrap',
+            -module     => 'Bio::EnsEMBL::Production::Pipeline::Webdatafile::WebdataFile',
+            -parameters => { current_step       => 'bootstrap',
+                             step => $self->o('step',)
```

Comment:
Typo here, "," should be outside the o('')

Reply:
resolved